### PR TITLE
Add in manual testing for add and delete lesson

### PR DESCRIPTION
**Pull request by laxus2308**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -801,17 +801,92 @@ testers are expected to do more *exploratory* testing.
...
+       Expected: No lesson is added. Error details are shown in status message.
+
+    4. Other incorrect add mastery check commands to try: `addstudio sd/2022-12-12`, `addstudio sd/2022-12-12 st/10:00`, `addstudio ed/`, `...`<br>
+       Expected: Similar to previous.
+
+### Deleting a Lesson
+
+1. Deleting a lesson while lesson list is being shown
+
+    1. Prerequisites: List all tasks using the `listlesson` or `listall` command. There are existing lessons in the lesson list.
```
Comment (rachaelsng) [inline docs/DeveloperGuide.md:843]:
```suggestion
    1. Prerequisites: List all lessons using the `listlesson` or `listall` command. There are existing lessons in the lesson list.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -769,9 +769,42 @@ testers are expected to do more *exploratory* testing.
...
+    1. Prerequisites: List all students using the `liststudent` or `listall` command. There must be at least one student in the list.
+   
+    2. Test case: `findstudent yeoh` <br>
+       Expected: Students' whose name contains `yeoh` regardless of capitalisation will be shown in the student list. Number of students listed is shown in the status message.
+   
+    3. Test case: `findstudent` <br>
+       Expected: No student is found. Error details shown in the status message.
+   
+    4. Test case: `findstudent x` (where x does not exist in any students' name in student list) <br> 
+       Expected: No student is found. Student list will be empty. Zero students listed is shown in the status message.
```
Comment (rachaelsng) [inline docs/DeveloperGuide.md:803]: I think can just stick to `Error details shown in the status message.` for consistency

Comment (laxus2308) [reply docs/DeveloperGuide.md:803]: For this, it isnt an error, is really no students found and blank list

**Review by rachaelsng — DISMISSED** [review]
Can take a look at the comment I wrote earlier. Otherwise, LGTM

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (rachaelsng) [reply docs/DeveloperGuide.md:803]: That makes sense

**Review by fongyj — APPROVED** [review]

**Pull request merged**